python/ray/data/block.py

```diff
@@ -118,7 +118,7 @@ def __call__(self, __arg: T) -> U:
     # UDF type.
     # Callable[[DataBatch, ...], DataBatch]
     Callable[[DataBatch], DataBatch],
-    _CallableClassProtocol,
+    "_CallableClassProtocol",
```

Comment:
This is needed for Py 3.6 compatibility as pickling will otherwise fail due to an error in typing extensions